modules/ROOT/pages/hybrid-schedule-mgmt.adoc

```diff
@@ -1,37 +1,99 @@
+= Manage Schedules
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
+// :page-aliases: hybrid-schedule-mgmt-config.adoc
 
 image:logo-cloud-disabled.png[link="/runtime-manager/deployment-strategies#cloudhub", title="CloudHub"]
 image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies#hybrid", title="Hybrid"]
 image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
-image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"
+image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
 
 
+You can use Runtime Manager to view and control the Scheduler components within the flows in your deployed applications.
+Use the *Schedules* tab in the left navigation menu to enable or disable Scheduler elements,
+change the frequency, or run the scheduled job immediately,
+all without changing the running application.
+
 [NOTE]
 Schedule management is supported on servers in a hybrid deployment environment using only Mule version 3.9 or later.
 
-You can use Runtime Manager to view and control polling elements within the flows of your deployed applications. Runtime Manager automatically detects the polling elements within your flows. These elements are listed on the *Schedulers* tab of the *Manage Applications* page.
-
-Runtime Manager lists the name of each poll element as `polling://{flow_name}`.
+From the *Schedules* tab, you can:
 
-From this tab, you can perform the following:
+* Enable or disable the Scheduler
+* Change the frequency
+* Specify when to start running the Scheduler
+* Specify the timezone (for a cron Scheduler)
 
-* Enable or disable polling elements.
-* Change the polling frequency.
 
-You can perform these tasks at runtime, without having to make changes to or redeploy the application.
+Runtime Manager lists the name of each Scheduler component as `polling://{flow_name}`.
 
 == Use Cases
 
 Schedule management is useful in the following contexts:
 
-* You want to change your schedules in your application at runtime without redeploying.
+* You want to change the schedules in your application at runtime without redeploying.
 * You want to disable a scheduled job while one of the applications you are connecting to is undergoing maintenance.
 * You want runtime visibility into the schedules in your application.
 
 
+== Considerations and Limitations
+
+When using the *Schedules* tab, consider the following:
+
+* If a scheduled job is not triggered because the application is not running,
+Runtime Manager triggers the job as soon as the application restarts.
+* When a schedule is used to trigger a flow with subflow in a Mule 4.x application, it waits until both main flow and subflow complete before scheduling the next run.
++
+*Is this true? I found it in the CloudHub Schedule info.*
```

Comment:
No sure about this, we need to validate it.

Reply:
Removed for now